src/main/java/frc/robot/subsystems/IntakeNote/IntakeNoteIO.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.subsystems.IntakeNote;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface IntakeNoteIO {
+  @AutoLog
+  class IntakeNoteIOInputs {
+    // I/p for intake
+    public boolean seesNote = false; // Need to communicate with Vision
+    public boolean armSwitchClosed = false;
+
+    // O/p for intake
```

Comment:
???

Reply:
These were from earlier in build season and were just serving as a thought. I will get rid of them, thank you for telling me.